src/Dialog/Dialog.js

```diff
@@ -161,8 +152,6 @@ Dialog.propTypes = {
     actions: PropTypes.arrayOf(PropTypes.node).isRequired,
     /** Localized text for the heading */
     title: PropTypes.string.isRequired,
-    /** CSS class(es) to add to the dialog backdrop */
-    backdropClassName: PropTypes.string,
```

Comment:
I think we should still keep the ability to put a class on the backdrop even though `fd-overlay` was removed